benchmarks/linear_algebra/blas/level1/copy/copy_generator.cpp

```diff
@@ -0,0 +1,60 @@
+
+#include <tiramisu/tiramisu.h>
+#include "benchmarks.h"
+
+/*
+  this is an implementation of the BLAS COPY (copies a vector to another vector) function in Tiramisu 
+  the performed operation is :
+   a = x
+
+   a : size N output vector
+   x : size N input vector
+   
+   The C version of this function is as follow : 
+   
+    for(int i=0; i<N:i++){
+          a(i)=x(i)
+    }
+   
+*/
+
+
+
+using namespace tiramisu;
+
+int main(int argc, char **argv)
+{
+    tiramisu::init("copy");
+
+    // -------------------------------------------------------
+    // Layer I
+    // -------------------------------------------------------
+    constant DIM("N", expr(N));
+    input x("x", {"i"}, {DIM}, p_float32);
+    var i("i", 0, N);
+    computation COPY("copy", {i}, (x(i)));
+
+    // -------------------------------------------------------
+    // Layer II
+    // -------------------------------------------------------
+    
+    COPY.vectorize(i, 32 );    
+
+
+    // -------------------------------------------------------
+    // Layer III
+    // -------------------------------------------------------
+
+    buffer b_x("b_x", {expr(DIM)}, p_float32, a_input);
+    buffer b_a("b_a", {expr(DIM)}, p_float32, a_output);
+    x.store_in(&b_x);
+    COPY.store_in(&b_a);
+
+    // -------------------------------------------------------
+    // Code Generation
+    // -------------------------------------------------------
+
+    tiramisu::codegen({&b_a, &b_x}, "generated_copy.o");
+
+    return 0;
+}
```

Comment:
Thanks! The code looks good to me, but I have one comment. The code does not respect the Tiramisu coding style. For example, the code should not have a new line when not necessary. There should not be a "{" when not necessary.  In

S1.set_expression( a(i) + b(i) );

There should not be a space before a(i) and after b(i).

These are all a part of the Tiramisu coding style. Can you please look at the Tiramisu codes and follow the same coding style so that all the codes are coherent?

Many thanks,

Reply:
Thanks for the feedback, I've fixed the coding style, is it okay like this?